15/umbraco-cms/reference/routing/iisrewriterules.md

```diff
@@ -128,6 +144,49 @@ Another example would be to redirect from non-www to www (except for the Umbraco
 </rule>
 ```
 
-{% hint style="warning" %}
-If you use **Umbraco Cloud** check the [Rewrite Rules](https://docs.umbraco.com/umbraco-cloud/set-up/project-settings/manage-hostnames/rewrites-on-cloud) article.
+### Example: Remove the .aspx Extension
+
+If you want to redirect .aspx URLs to their extensionless counterparts, you can use the following rule. Make sure you also have a web.config file in the root of your site:
```

Comment:
I'm not sure the sentence about the `web.config` is necessary, as all the examples above also need to go into the `web.config` file, right? 🤔 

```suggestion
The following rule redirects `.aspx` URLs to their extensionless counterparts. Make sure you also have a `web.config` file in the root of your site.
```

Reply:
Yes, you are right. I missed it was mentioned in the Note above.